$NetBSD: patch-src_platform_qt_CMakeLists.txt,v 1.3 2022/12/15 11:31:44 nia Exp $

Mark Multimedia an optional component so we can build without the
Qt5 audio driver (using only SDL).

--- src/platform/qt/CMakeLists.txt.orig	2022-10-12 03:30:00.000000000 +0000
+++ src/platform/qt/CMakeLists.txt
@@ -25,7 +25,7 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 set(QT_LIBRARIES)
 
-find_package(Qt5 COMPONENTS Core Widgets Network Multimedia)
+find_package(Qt5 COMPONENTS Core Widgets Network OPTIONAL_COMPONENTS Multimedia)
 set(QT Qt5)
 
 if(NOT BUILD_GL AND NOT BUILD_GLES2 AND NOT BUILD_GLES3)
